app/views/components/feedback_section.scala.html

```diff
@@ -0,0 +1,29 @@
+@*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import config.ExternalServicesConfig
+
+@this(config: ExternalServicesConfig)
+
+@()(implicit messages: Messages)
+
+@link = {<a href="@config.feedbackFrontend">@messages("feedback.link")</a>}
+
+<div>
```

Comment:
is this div doing anything? It seems unnecessary additional markup to me

Reply:
Good point. I have removed it.